src/UI/EditorView.re

```diff
@@ -248,11 +248,16 @@ let make =
       <Parts.EditorContainer editor state theme isActive=true dispatch />;
   };
 
+  let hideZenModeTabs =
+    state.configuration |> Configuration.getValue(c => c.zenModeHideTabs);
+
   <View onFileDropped style={Styles.container(theme)}>
     <View style={Styles.container(theme)}>
       <Feature_Layout.View
         uiFont={state.uiFont}
         theme
+        isZenMode={state.zenMode}
+        shouldShowTabsInZenMode={!hideZenModeTabs}
```

Comment:
Do we need layout to know about zen mode? Would it not be better to just have `showTabs`?

There's also the `workbenchEditorShowTabs` configuration setting.

Reply:
Good catch @glennsl ! You're right, would be better just to have `showTabs`. Switched to this in https://github.com/onivim/oni2/pull/1989/commits/e78992fa43bac1791be678b3240cee1f0a806b14